cmd/lncli/cmd_bake_macaroon.go

```diff
@@ -48,6 +48,10 @@ var bakeMacaroonCommand = cli.Command{
 			Name:  "ip_address",
 			Usage: "the IP address the macaroon will be bound to",
 		},
+		cli.Uint64Flag{
```

Comment:
nit: the changes to `lncli` should probably go into a separate commit. Then the commit messages can be prefixed with `lnrpc+rpcserver: ` and `lncli: ` respectively.